lib/manageiq/messaging/kafka/common.rb

```diff
@@ -116,7 +116,7 @@ def message_header_keys
 
         def parse_message_headers(headers)
           return [nil, nil, nil] unless headers.kind_of?(Hash)
-          headers.values_at(*message_header_keys)
+          headers.with_indifferent_access.values_at(*message_header_keys)
```

Comment:
We should add a `require "active_support/core_ext/hash/indifferent_access"` to this class to ensure we have access to this extension (this being a gem and not a rails app or engine activesupport being pulled in isn't a given and you have to require specifically what you need)